backend/src/eml/tests/deserialize_eml510b_test.eml.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<EML Id="510b" SchemaVersion="5" xmlns="urn:oasis:names:tc:evs:schema:eml" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:kr="http://www.kiesraad.nl/extensions" xmlns:rg="http://www.kiesraad.nl/reportgenerator" xmlns:xal="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0" xmlns:xnl="urn:oasis:names:tc:ciq:xsdschema:xNL:2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <CanonicalizationMethod Algorithm="http://www.w3.org/TR/2001/REC-xml-c14n-20010315#WithComments"/>
+    <TransactionId>1</TransactionId>
+    <ManagingAuthority>
+        <AuthorityIdentifier Id="1">Test Authority</AuthorityIdentifier>
+        <AuthorityAddress/>
+    </ManagingAuthority>
+    <kr:CreationDateTime>2021-09-01T12:00:00</kr:CreationDateTime>
+    <Count>
+        <EventIdentifier/>
+        <Election>
+            <ElectionIdentifier>
+                <ElectionName>Municipal Election</ElectionName>
+                <ElectionCategory>GR</ElectionCategory>
+                <kr:ElectionSubcategory>GR1</kr:ElectionSubcategory>
+                <kr:ElectionDomain Id="1">Gemeente</kr:ElectionDomain>
+                <kr:ElectionDate>2021-09-01</kr:ElectionDate>
+            </ElectionIdentifier>
+            <Contests>
+                <Contest>
+                    <ContestIdentifier Id="geen"/>
+                    <TotalVotes>
+                        <Selection>
+                            <AffiliationIdentifier Id="1">
+                                <RegisteredName>Test Party</RegisteredName>
+                            </AffiliationIdentifier>
+                            <ValidVotes>100</ValidVotes>
+                        </Selection>
+                        <Selection>
+                            <Candidate>
+                                <CandidateIdentifier Id="1"/>
+                            </Candidate>
+                            <ValidVotes>50</ValidVotes>
+                        </Selection>
+                        <Selection>
+                            <Candidate>
+                                <CandidateIdentifier Id="2"/>
+                            </Candidate>
+                            <ValidVotes>50</ValidVotes>
+                        </Selection>
+                        <Cast>100</Cast>
+                        <TotalCounted>100</TotalCounted>
+                        <RejectedVotes ReasonCode="ongeldig">0</RejectedVotes>
+                        <RejectedVotes ReasonCode="blanco">0</RejectedVotes>
+                    </TotalVotes>
+                    <ReportingUnitVotes>
+                        <ReportingUnitIdentifier Id="1">Op rolletjes</ReportingUnitIdentifier>
```

Comment:
Schema validation error: `deserialize_eml510b_test.eml.xml:48: element ReportingUnitIdentifier: Schemas validity error : Element '{urn:oasis:names:tc:evs:schema:eml}ReportingUnitIdentifier', attribute 'Id': [facet 'pattern'] The value '1' is not accepted by the pattern '(HSB\d+)|((HSB\d+::)?\d{4})|(((HSB\d+::)?\d{4}::)?SB\d+)|(HSB\d+::SB\d+)'.`